src/web/modules/transactions/fix_failed_stripe_refund.ts

```diff
@@ -0,0 +1,34 @@
+import {NextFunction, Request, Response} from "express";
+import {Connector, Ledger} from "../../../lib/pay-request/client";
+
+export async function show(req: Request, res: Response, next: NextFunction): Promise<void> {
+    try {
+        const transaction = await Ledger.transactions.retrieve(req.params.id)
+
+        res.render(`transactions/refundConfirmation`, {
```

Comment:
Again, I think we want to be precise here so maybe `confirmFixAsyncFailedStripeRefund`.